x-pack/plugins/canvas/tasks/test.js

```diff
@@ -7,6 +7,7 @@
 import { resolve, join } from 'path';
 
 export default function testTasks(gulp, { mocha }) {
+  require('./mocks/canvas_i18n');
```

Comment:
nit: This might be better wrapped up in an enable/disable function. Then we can just import it like normal, enable it before running the tests, and disable if we ever need to use the non-stubbed instance in tests.